ci/verify_current_targets/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 # ~~~
 
-cmake_minimum_required(VERSION 3.6)
```

Comment:
hmm

Reply:
I am not sure what you are pondering, but FWIW, this was used in a single CI build that provided CMake > 3.5 anyway.

Reply:
Was he not pondering that the CHANGELOG says "CMake >= 3.10"?

Reply:
The "hmm" was because this one was a `3.6` and all the others were `3.5`.

I just notice patterns. I do not reason about what things mean.